app/models/user.js

```diff
@@ -1,16 +1,64 @@
 'use strict';
+<<<<<<< HEAD
+
+const bcrypt = require('bcrypt-nodejs');
+
+module.exports = function(sequelize, DataTypes) {
+=======
 module.exports = (sequelize, DataTypes) => {
+>>>>>>> master
   var user = sequelize.define('users', {
-    name: DataTypes.STRING,
-    lastName: DataTypes.STRING,
-    email: DataTypes.STRING,
-    password: DataTypes.STRING
+    name: {
+      type: DataTypes.STRING, 
+      allowNull: false,
+      notEmpty: true
+    },
+    lastName: {
+      type: DataTypes.STRING, 
+      allowNull: false,
+      notEmpty: true
+    },
+    password: {
+      type: DataTypes.STRING, 
+      allowNull:false,
+      notEmpty: true,
+      validate: {
```

Comment:
You are not validating that the password only contains alfanumeric chars.
This validations are checked when we edit a user?